Orange/data/filter.py

```diff
@@ -379,16 +375,11 @@ def __str__(self):
             return "{} is defined".format(column)
         return "invalid operator"
 
-    __repr__ = __str__
-
-
-    # For PyCharm:
-    Equal = NotEqual = Less = LessEqual = Greater = GreaterEqual = 0
```

Comment:
I guess this was here to allow IDEs to suggest these names for code completion. Does this work without the dummy members in the current version of PyCharm?